wire-ios/Wire-iOS/Sources/Components/GridLayoutView.swift

```diff
@@ -27,7 +27,7 @@ public final class GridLayoutView: UIView {
 
     // MARK: - Properties
 
-    var verticalSpacing: CGFloat = 0
+    var verticalSpacing: CGFloat = 4
```

Comment:
@johnxnguyen should it be `4` or `0`?

Reply:
4 is correct, I forgot to change it back.